peachjam/models/core_document_model.py

```diff
@@ -169,6 +169,10 @@ class Work(models.Model):
     )
     # the rank (weight) of this work in the graph network, computer by peachjam.graph.ranker
     ranking = models.FloatField(_("ranking"), null=True, blank=False, default=0.0)
+    # number of outgoing citations
+    n_cited_works = models.IntegerField(_("number of cited works"), default=0)
+    # number of incoming citations
+    n_citing_works = models.IntegerField(_("number of incoming citations"), default=0)
```

Comment:
I was worried about this slowing down the pages with many citing documents but this solves that :)